src/js/components/covid19/Heading.jsx

```diff
@@ -4,20 +4,31 @@
  */
 
 import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import GlossaryLink from 'components/sharedComponents/GlossaryLink';
+import ExternalLink from 'components/sharedComponents/ExternalLink';
+import ReadMore from './ReadMore';
 
 const Heading = () => (
     <div className="heading__container information-body">
         <div className="heading__top information-top" />
         <div className="heading__title">
-            How much has the Federal Government
-            spent on the <span className="color-purple">COVID-19 Response</span> <strong>in total</strong>?
+            The Federal Response to <span className="color-purple">COVID-19</span>.
         </div>
         <div className="heading__description">
-            In April 2020 the Federal Government began allocating funds in
-            response to the COVID-19 pandemic. There are a variety of
-            dimensions to analyze COVID-19 Response spending,
-            including who received spending, which agency spent the funds, and how
-            the funds were spent over time.
+            <p>In early 2020, the U.S. Congress appropriated funds in response to the COVID-19 pandemic. These funds were made possible through the Coronavirus Aid, Relief, and Economic Security (CARES) Act and other supplemental legislation.</p>
+            <ReadMore>
+                <p>
+                    In response to guidance from the Office of Management and Budget (OMB), we updated our data model to capture the journey of COVID-19 dollars from appropriation to <span className="glossary-term">obligation</span> <GlossaryLink currentUrl="disaster/covid-19" term="obligation" /> and <span className="glossary-term">outlay</span> <GlossaryLink currentUrl="disaster/covid-19" term="outlay" /> by federal agencies.<sup>1 2</sup> See who received funding, which agencies spent the funds, which programs were funded, and more.
+                </p>
+                <p>
+                    Visit our Data Sources & Methodology and Other Resources sections to learn more about the underlying data and find resources about COVID-19 from other agencies.
```

Comment:
Looks like these (Data Sources & Methodology, Other Resources) are jump to sections, not sure if the mock changed.
<img width="820" alt="Screen Shot 2020-07-29 at 2 31 32 PM" src="https://user-images.githubusercontent.com/14301977/88839838-99e7b980-d1a9-11ea-9196-37c31ca02692.png">